library/dialpad/build.gradle

```diff
@@ -0,0 +1,26 @@
+apply plugin: 'com.android.library'
+apply plugin: 'kotlin-android'
+
+android {
+    compileSdk versions.compileSdk
+
+    defaultConfig {
+        minSdk versions.minSdk
+        targetSdk versions.targetSdk
+    }
+
+    compileOptions {
+        sourceCompatibility versions.sourceCompat
+        targetCompatibility versions.targetCompat
+    }
+
+    kotlinOptions {
+        jvmTarget = "11"
+    }
+}
+
+afterEvaluate {
+    tasks.findAll { it.name.startsWith("lint") }.each {
+        it.enabled = false
```

Comment:
lint is disabled here, because there are too many issues reported?

Reply:
I wasn't sure on our stance, do we want to fix lint issues from external projects?

Reply:
I can't remember the exact amount, but there were lots of lint issues about API versions, unused resources, etc. Since it's not really our code (we're just forking it to solve the mentioned issues) fixing all of those didn't seem to be worth the effort.

Reply:
This is maybe not clear enough for everyone that this is external project, since it's in the `library` folder. Maybe create another folder like `library_external`, or a subfolder `library/external`.
But I agree, that we do not want to spend time to fix issues on all our dep.

Reply:
sounds good :+1:  Should I move `jsonviewer` as well? as I guess that's technically originates from an external repository~

Reply:
yes, please, and `diff_match_patch` too.

Reply:
a9298c7c0354d1926b057484028ff92dca68a245 moves the modules to `library/external`